crates/red_knot/src/main.rs

```diff
@@ -34,63 +34,48 @@ struct Args {
     #[command(subcommand)]
     pub(crate) command: Option<Command>,
 
-    #[arg(
-        long,
-        help = "Changes the current working directory.",
-        long_help = "Changes the current working directory before any specified operations. This affects the workspace and configuration discovery.",
-        value_name = "PATH"
-    )]
-    current_directory: Option<SystemPathBuf>,
-
-    #[arg(
-        long,
-        help = "Path to the virtual environment the project uses",
-        long_help = "\
-Path to the virtual environment the project uses. \
-If provided, red-knot will use the `site-packages` directory of this virtual environment \
-to resolve type information for the project's third-party dependencies.",
-        value_name = "PATH"
-    )]
+    /// Run the command within the given project directory.
+    ///
+    /// All `pyproject.toml` files will be discovered by walking up the directory tree from the project root,
```

Comment:
I find this sentence a bit confusing, because I thought the "project root" is defined by where we find a `pyproject.toml` with the expected config? It might make more sense to phrase this as something like "The project root will be found relative to this directory" or similar?

Reply:
Fair, I copied this from uv 🤭